tests/h/search/query_test.py

```diff
@@ -217,11 +217,43 @@ def search(self, search):
 
 
 class TestRepliesMatcher(object):
+    def test_matches_unnested_replies_to_annotations(self, Annotation, search):
+        ann1 = Annotation()
+        ann2 = Annotation()
+        ann3 = Annotation()
+        Annotation()
+        # Create two replies on ann1.
+        reply1 = Annotation(references=[ann1.id])
+        reply2 = Annotation(references=[ann1.id])
+        # Create a reply on ann2
+        reply3 = Annotation(references=[ann2.id])
+        # Create a reply on ann3
+        Annotation(references=[ann3.id])
+
+        expected_reply_ids = [reply1.id, reply2.id, reply3.id]
+
+        ann_ids = [ann1.id, ann2.id]
+        search.append_matcher(query.RepliesMatcher(ann_ids))
+        result = search.run({})
+
+        assert sorted(result.annotation_ids) == sorted(expected_reply_ids)
+
+    def test_matches_replies_of_replies_to_an_annotation(self, Annotation, search):
+        ann1 = Annotation()
```

Comment:
These three lines (`ann1 =...`, `# Create a reply...`, `reply1 = ...`) are duplicates of the lines below.